test-clients/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
The maven wrapper has been removed from test-clients module. However, the `cd test-clients` command still needs to be present, since E2E tests need to access resources from test-clients in order to execute successfully.